tests/runtime/utils/test_servicer_util.py

```diff
@@ -520,145 +439,5 @@ def test_build_caikit_library_request_dict_works_when_data_stream_directory_incl
         # no error because at least 1 json file exists within the provided dir
         caikit.core_request = build_caikit_library_request_dict(
             train_request,
-            sample_lib.modules.sample_task.SampleModule().train,
-        )
-
-
-# ---------------- Error tests for build_caikit_library_request_dict --------------------
```

Comment:
Removing these error tests that depend on `to_data_stream`. These errors will occur not at request building time but at request usage time